ComfortZoneApp/app/src/main/java/com/example/comfortzone/ui/LoadingFragment.java

```diff
@@ -0,0 +1,38 @@
+package com.example.comfortzone.ui;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.comfortzone.R;
+
+public class LoadingFragment extends Fragment {
```

Comment:
Can you rename this to be more meaningful? Different screens can have different loading fragments. If this is for the HostActivity, you can rename this to be HostLoadingFragment?